src/js/components/MainDiv.jsx

```diff
@@ -108,53 +109,39 @@ const navbarPagesFn = () => {
 	return [...pages];
 };
 
+/**
+ * navbar search widget
+ */
 const SearchWidget = () => {
-    const searchIcon = <Misc.Icon prefix="fas" fa="search" />;
-
-    const searchQuery = getValue([...SEARCH_QUERY_DATASTORE_PATH, SEARCH_QUERY_DATASTORE_PROP]);
-    const onSubmit = (e) => {
-        DataStore.setUrlValue("q", searchQuery);
-    };
-
-    const url = '#search?q=' + DataStore.getUrlValue('q');
-    const submitButton = (
-        <a href={url}>
-            <Button
-                type="submit"
-                onClick={onSubmit}
-                color="primary"
-                className="sogive-search-box"
-            >
-                Search
-            </Button>
-        </a>
-    );
-
-    return (
-        // Unable to use a Form here because it prevents the submitButton navigating to the
-        // Search page when the user is on the Charity page (the PropControl prop gets appended
-        // to the URL before the '#search' page anchor). (Unfortunately this means the user
-        // can't press 'Enter' in the textbox to search, they must click the submit button.)
-        <div className="navbar-search-widget ml-auto">
-            <PropControl
-                path={SEARCH_QUERY_DATASTORE_PATH}
-                prop={SEARCH_QUERY_DATASTORE_PROP}
-                type="search"
-                placeholder="Enter a charity's name"
-                prepend={searchIcon}
-                append={submitButton}
-                size="lg"
-            />
-            <FieldClearButton />
-        </div>
-    );
+	// NB: PropControl or Input + useState? 
+	// PropControl is best if the data is shared with other components, or if the data should be maintained across page changes, or if extras like help and error text are wanted. 
+	// useState is best for purely local state.
+	let [q, setQ] = useState("");
+	const onSubmit = (e) => {
+		stopEvent(e);
+		modifyHash(['search'], {q});
+	};
+
+	return (
+		<div className="navbar-search-widget ml-auto">
+			<form onSubmit={onSubmit}>
```

Comment:
curious what's the reason to use an html 'form' instead of reactstrap 'Form' here?

Reply:
A mild preference for vanilla no-dependency widgets when they make sense. I don't think `Form` does anything more than `form`.